From: Richard Neuboeck <hawk@tbi.univie.ac.at>
Subject: Fixes SAS sysfs path lookup

--- a/util/ieee1275/ofpath.c	2012-11-05 19:22:38.319236924 +0100
+++ b/util/ieee1275/ofpath.c.new	2012-11-05 19:23:20.593413652 +0100
@@ -337,9 +337,9 @@
   *q = '\0';
 
   path_size = (strlen (p) + strlen (ed)
-	       + sizeof ("%s/sas_device:%s/phy_identifier"));
+	       + sizeof ("%s/sas_device/%s/phy_identifier"));
   path = xmalloc (path_size);
-  snprintf (path, path_size, "%s/sas_device:%s/phy_identifier", p, ed);
+  snprintf (path, path_size, "%s/sas_device/%s/phy_identifier", p, ed);
 
   fd = open (path, O_RDONLY);
   if (fd < 0)
